po/da.po

```diff
@@ -4390,9 +4380,8 @@ msgstr ""
 "```"
 
 #: src/ownership/double-free-modern-cpp.md:1
-#, fuzzy
 msgid "Extra Work in Modern C++"
-msgstr "Dobbeltfrigivelser i moderne C++"
+msgstr "Ekstra arbejde i moderne C++"
```

Comment:
I'm not sure I have a better translation but I like that its not clear what kind of work we're talking about without additional context. It is not the programmer who has to do more work here; it is the machine. If we can change the English text, I'd use "Additional duplication in modern C++"; if not, I'd translate that to Danish for additional clarity.

OTOH, I don't know if you're comfortable with the additional "distance" between the English and Danish wording.

Reply:
> If we can change the English text, I'd use "Additional duplication in modern C++"; if not, I'd translate that to Danish for additional clarity.

We can (and _should!_) update the English text to fix this kind of awkwardness. There are lots of text like this which can be expanded just a little to make it easier to understand in both English and other languages.
 
> OTOH, I don't know if you're comfortable with the additional "distance" between the English and Danish wording.

I'm not an expert, but my gut feeling would be to minimize the distance. In this case, I will try to fix both at the same time: I can update the English text (I created #1192) and I can preemptively translate the new heading. It will be marked fuzzy when the PR is merged and then I can proof-read it again.

Reply:
```suggestion
msgstr "Defensiv kopiering i moderne C++"
```